Valet/VALValet.m

```diff
@@ -450,6 +450,10 @@ - (BOOL)setObject:(nonnull NSData *)value forKey:(nonnull NSString *)key options
             keychainData[(__bridge id)kSecValueData] = value;
             
             status = VALAtomicSecItemAdd((__bridge CFDictionaryRef)keychainData, NULL);
+            
+            if (status == errSecMissingEntitlement) {
```

Comment:
Just occurred to me that this error could occur for legitimate reasons (see #2 in the Debugging section of the readme). If this throws anyone for a loop I'll update the log to point to the Debugging section of the readme